magstock/templates/emails/reg_workflow/attendee_confirmation.html

```diff
@@ -0,0 +1,42 @@
+<html>
```

Comment:
There were enough changes here that we decided to just override the entire template.